src/SonarScanner.Shim/PropertiesWriter.cs

```diff
@@ -31,10 +31,12 @@ namespace SonarScanner.Shim
 {
     public class PropertiesWriter
     {
+        private readonly ILogger logger;
         private readonly StringBuilder sb;
-
         private readonly AnalysisConfig config;
 
+        private static readonly string MultiValuesPropertySeparator = $@",\{Environment.NewLine}";
```

Comment:
Not sure whether it should be a field or local variable now that I extracted the behavior to a single method.

Reply:
If it's localised to a single method then a local variable might be better.